src/ca/client/netiiu.cpp

```diff
@@ -116,6 +116,12 @@ const char * netiiu::pHostName (
     return pHostNameNetIIU;
 }
 
+unsigned netiiu::getHostMinorProtocol ( 
+    epicsGuard < epicsMutex > & ) const throw ()
+{
+    return 0u;
+}
```

Comment:
Why implement this pure virtual method?

If this is meant to be a default, wouldn't `CA_MINOR_PROTOCOL_REVISION` be more appropriate?  Or `cantProceed()` if it is not meant to be called?

Reply:
I coded this method exactly parallel to how getHostName() and getNetworkAddress() are done. They are all declared pure virtual, but then defined.